src/main/java/com/gregtechceu/gtceu/client/mui/screen/RichTooltip.java

```diff
@@ -171,6 +171,8 @@ public void draw(GuiContext context, @Nullable ItemStack stack) {
         RenderSystem.disableDepthTest();
         RenderSystem.disableBlend();
 
+        context.getGraphics().pose().pushPose();
+        context.getGraphics().pose().translate(0, 0, 400);
```

Comment:
how come the magic number translate came back :(

Reply:
While it does "work" in most cases, this is for safety. Mc also translates by 400 in their tooltip method, so i thought we should do the same.